multiuser/machine-auth/che-multiuser-machine-authentication-agent/src/main/java/org/eclipse/che/multiuser/machine/authentication/agent/MachineLoginFilter.java

```diff
@@ -84,7 +84,7 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
     try {
       final UserDto userDescriptor =
           requestFactory
-              .fromUrl(tokenServiceEndpoint + "/user/" + machineToken)
+              .fromUrl(apiEndpoint + "/user/")
```

Comment:
I think now you should change catching of `NotFoundException` to catching of `UnauthorizedException`